### PR TITLE
Remove/redirect open source contribution methods on China site

### PR DESCRIPTION
**Pull request by smallst**
https://app.asana.com/0/694028898835437/1129644507505018/f

### CODE REVIEW

File: app/templates/about.jade [hunk 1]
```diff
@@ -22,9 +22,10 @@ block content
             li
               a(href="#team")#team-link
                 small.label(data-i18n="about.team_link")
-            li
-              a(href="#community")#community-link
-                small.label(data-i18n="nav.community")
+            if !features.china
```
Comment (wjllance) [inline app/templates/about.jade]: better create a new method like `showOpenResouceLink()` in User model

**Comment by smallst** [comment]
there are 4 places changed:
about page:
about-git
front page footer:
fp-git
legal page: (i don't know if code MIT license show hide too)
legal-git
user settting page:
setting-git

**Comment by wjllance** [comment]
image


please remove this two buttons too, thanks

**Comment by wjllance** [comment]
image

**Pull request merged**